### PR TITLE
Reorder and improve Tensorboard tips.

### PR DESCRIPTION
**Pull request by lesteve**
Some reordering and improvement on the Tensorboard section.

I would be curious if one of you encountered cases where your `sshfs` gets disconnected (e.g. `ls` on your mounted PATH hangs for ever).

Ages ago (not on Jean Zay) I seem to remember I had this problem from time to time and even sometimes I was unable to unmount the folder with `fusermount -u`. Maybe I did not know about `-o reconnect,ServerAliveInterval=15,ServerAliveCountMax=3` at that time though.

### CODE REVIEW

**Comment by remicres** [comment]
> I would be curious if one of you encountered cases where your `sshfs` gets disconnected (e.g. `ls` on your mounted PATH hangs for ever).

I sometimes have to umout/remount after a maintenance.

**Comment by RemiLacroix-IDRIS** [comment]
@lesteve : Would it be possible not to use the full path for the `$SCRATCH` in your example? I understand it makes things a bit complex but you could ask the user to copy-paste the path obtained from `pwd` on Jean Zay or something like that.

Also, keep in mind that sshfs is not officially supported on Jean Zay, although I don't think anything will change regarding the possibility to use SSHFS.

**Comment by lesteve** [comment]
> Also, keep in mind that sshfs is not officially supported on Jean Zay, although I don't think anything will change regarding the possibility to use SSHFS.

OK I am going to read this as it will be possible to use sshfs in the future, but don't ask the Jean Zay support if you have a problem with it 😉.

Also I tweaked the folder name.

**Comment by RemiLacroix-IDRIS** [comment]
The hotline might be able to help but we cannot promise that SSHFS will still be possible tomorrow, in a week, a month or a year from now (even if I personally don't expect any changes).

> Also I tweaked the folder name.

Thanks, if you could squash the commits that would be perfect.

**Comment by mdiazmel** [comment]
> I would be curious if one of you encountered cases where your sshfs gets disconnected (e.g. ls on your mounted PATH hangs for ever).

> Ages ago (not on Jean Zay) I seem to remember I had this problem from time to time and even sometimes I was unable to unmount the folder with fusermount -u. Maybe I did not know about -o reconnect,ServerAliveInterval=15,ServerAliveCountMax=3 at that time though.

I have had this problem (not frequently), with some exotic setups, where I need to use VPN to access the ssh server (not in Jean Zay). For example, the mount point hang forever when I lost the VPN connection and it reconnects later. In this particular case, the option `-o reconnect` doesn't seem to work properly.  To avoid this, I used to mount/unmount the sshfs partition only the time that I need to access the data.

In the same sense of this PR, I think that it should be nice to reorder a the "Tips and Ticks" section. It's becoming a little messy!

File: docs/tips-and-tricks.md [hunk 1]
```diff
@@ -266,59 +266,86 @@ http://www.idris.fr/media/data/formulaires/fgc.pdf, mostly the section "Ajout,
...
+  of Jean-Zay
+- use `sshfs` so that your Tensorboard logs (that are on Jean-Zay) are visible
+  on your local machine
+- run Tensorboard locally on your logs mounted with `sshfs`
 
-An alternative to the official solution is to mount a folder from Jean
-Zay, where you store the logs generated by Tensorboard, and use
-the Tesorboard web server locally to visualize them. Let's suppose that 
-you store logs in `$SCRATH/tb_logs/`
+For example if your Tensorboard logs are in `$SCRATH/tb_logs/`
```
Comment (mdiazmel) [inline docs/tips-and-tricks.md:284]: Change this also: 

```suggestion
For example if your Tensorboard logs are in `/your/jean-zay/tensorboard/log/folder/`
```

**Comment by lesteve** [comment]
Merging this one, thanks for the comments!

> In the same sense of this PR, I think that it should be nice to reorder a the "Tips and Ticks" section. It's becoming a little messy!

Agreed cleaning it up a bit would be nice, maybe by broad categories similar to this:
- having a nice setup where you can work productively
- ssh-related tips
- SLURM-related stuff
- AI libraries hacks
- Python
- miscellaneous

maybe split it in multiple files. Not sure maybe wait a bit more until it becomes way too long before splitting it.

File: docs/tips-and-tricks.md [hunk 2]
```diff
@@ -266,59 +266,94 @@ http://www.idris.fr/media/data/formulaires/fgc.pdf, mostly the section "Ajout,
...
-2. Install tensorboard in your host machine:
+2. Install Tensorboard on your local machine for example with pip:
 ```
 pip install tensorboard
 ```
 
-3. Launch tensorboard locally, in the host machine. 
+3. Launch Tensorboard on your local machine
 ```
 tensorboard --logdir=$HOME/jeanzay/logs
```
Comment (remicres) [inline docs/tips-and-tricks.md:301]: `/your/local/tensorboard/log/folder` instead of `$HOME/jeanzay/logs`

**Review by remicres — APPROVED** [review]

File: docs/tips-and-tricks.md (hunk 2 above)
Comment (lesteve) [reply docs/tips-and-tricks.md:301]: right I'll fix that (after my next meeting)! Feel free to do it in the meantime 😉

Comment (lesteve) [reply docs/tips-and-tricks.md:301]: Done

**Pull request merged**